rocAL/include/loaders/circular_buffer.h

```diff
@@ -31,8 +31,8 @@ THE SOFTWARE.
 #include "commons.h"
 #include "device_manager.h"
 #include "device_manager_hip.h"
-struct decoded_image_info {
-    std::vector<std::string> _image_names;
+struct decoded_sample_info {
```

Comment:
please change the struct name to camel case. Instead of naming _sample_names, you can have a union which can hold image_names, video_file_names or audio_sample_name. That way individual reader can use appropriate names

Reply:
Struct name changed to camelCase. Working on the union part.